clients/cmd/fluentd/lib/fluent/plugin/out_loki.rb

```diff
@@ -278,6 +278,8 @@ def record_to_line(record)
           when :key_value
             formatted_labels = []
             record.each do |k, v|
+              # Remove non UTF-8 characters by force-encoding the string and replacing said chars with empty string
+              v = v.encode('utf-8', invalid: :replace, replace: '')
```

Comment:
should we replace with another special character may be ? Not sure if we can empty string will not tell users there was an issue there ? WDYT ?

Reply:
would � work ?

Reply:
You can let Ruby decide how to replace the invalid characters, like so:

`              v = v.encode('utf-8', invalid: :replace)`